src/corelibs/U2Formats/src/StockholmFormat.cpp

```diff
@@ -94,491 +75,331 @@ enum AnnotationTag {
     NC,
     TC
 };
-// other types not supported
+
+/** Set of supported annotation types. */
 enum AnnotationType {
     FILE_ANNOTATION,
     COLUMN_ANNOTATION,
     UNI_ANNOTATION
 };
 
-struct Annotation {
-    AnnotationType type;
-    AnnotationTag tag;
-    QString val;
-    Annotation(AnnotationType ty, AnnotationTag t, QString v) {
-        type = ty, tag = t;
-        val = v;
-    }
-    virtual ~Annotation() {
-    }
-};
-//#=GF annotations
-struct FileAnnotation : public Annotation {
-    FileAnnotation(AnnotationTag t, QString v)
-        : Annotation(FILE_ANNOTATION, t, v) {
+/** Internal representation of the annotation used internally during parsing. */
+class StockholmAnnotation {
```

Comment:
Please, do something about thisc class - split declarations/definitions and move to separate files

Reply:
Why? There is nothing bad in local API defined locally. 

All modern languages I know use his style and the archaic C/C++ way of declaration/definition separation is a solution for a different problem: the one we do not have in this case.